src/workerd/api/form-data.c++

```diff
@@ -273,7 +273,7 @@ kj::Array<FormData::EntryWithoutLock> FormData::parseWithoutLock(
         } else {
           data.add(FormData::EntryWithoutLock{
             .name = kj::str(name),
-            .value = kj::str(message),
+            .value = kj::heapArray<kj::byte>(message),
```

Comment:
Keeping it a string would preserve symmetric behavior with `FormData::parse()`, where multipart/form-data with no filename is exposed via the JS API as a string.

I realize it doesn't really make any difference for our use case, though.